### ISSUE TITLE
Fix debian package version

### ISSUE THREAD

**Issue by landryb** [georchestra/mapstore2-georchestra#463]
After #407 the produced debian package is supposed to have `georchestra-mapstore_<closest tag name>-<number of commits since the previous tag>_all.deb` for the name, but unfortunately it doesnt seem to fully work.

eg the artifacts generated on commits (since #448) only have the `<closest tag name>` in the version and thus it's impossible to upgrade them, as they all have exactly the same version.

```
$apt show georchestra-mapstore
Package: georchestra-mapstore
Version: 20.1.3-georchestra
```

i think the plumbing done in #407 should remove the `-georchestra` prefix from the tag (or use the branch name) and should ideally add the build or commit date and commit sha like other packages so that upgrades can move forward.

@pmauduit ?

**Comment by landryb** [comment]
i'm trying:
```
-                            <packageVersion>${build.closest.tag.name}</packageVersion>
-                            <packageRevision>${build.closest.tag.commit.count}</packageRevision>
+                            <packageVersion>${build.branch}.${maven.build.timestamp}~${build.commit.id.abbrev}</packageVersion>
```

**Comment by landryb** [comment]
ah, i think i have something that works and produces ` Version: 99.master.202201251523~7c61adb` on master branch and `Version: 2021.02.xx.202201251525~a99d420` on 2021.02.xx branch when the debian package is inspected with `dpkg --info`

That should align with what we use in georchestra project.. and allow for seamless debian pkg upgrades.

**Comment by landryb** [comment]
closed by #466 in stable branch
